astropy/table/tests/test_table.py

```diff
@@ -655,6 +655,14 @@ def test_add_duplicate_columns(self, table_types):
         t.add_column(self.d)
         assert t.colnames == ['a', 'b', 'c', 'a_1', 'b_1', 'd']
 
+    def test_add_list_order(self, table_types):
+        self._setup(table_types)
+        t = table_types.Table()
```

Comment:
For this test there is no additional coverage provided by the three different `table_types` (they all wind up at the same base method). There are likely other tests in this class where that is also true, but for now you should just move this out of the class as a single test and do `t = Table()`.

Reply:
Thanks for clarifying. I saw those three table types during the test, but had no idea what that is for or when a test should use it. I will update the PR.

The standalone test functions don't seem to have specific sort order, so I just put the function at the end. Please let me know if anything is wrong.